PATTERNS.md

```diff
@@ -66,7 +66,12 @@ The most robust and scalable way to accomplish this is by combining the standard
 Reusing Redis Connections
 -------------------------
 
-A standard queue requires **3 connections** to the Redis server. In some situations you might want to re-use connections—for example on Heroku where the connection count is restricted. You can do this with the `createClient` option in the `Queue` constructor (note: bclient connections [cannot be re-used](https://github.com/OptimalBits/bull/issues/880)):
+A standard queue requires **3 connections** to the Redis server. In some situations you might want to re-use connections—for example on Heroku where the connection count is restricted. You can do this with the `createClient` option in the `Queue` constructor.
+
+Notes:
+- bclient connections [cannot be re-used](https://github.com/OptimalBits/bull/issues/880), so you should return a new connect each time this is called. These connections will be disconnected/reconnected by bull automatically as part of queue operations and will be disconnected when the queue is closed
```

Comment:
A note here, the bclient is not disconnected when the queue closes. See it like this, all connections created outside of Bull are never closed or destroyed by Bull and need to be disconnected manually after closing Bull.

Reply:
Ah right, I was confused because it does kind of ... disconnect and reconnect all the time.  But after the queue is closed, the connection is left in a "connected" state, so it still has to be closed in a separate step.